### ISSUE TITLE
Make plurals fallback to those of locale fallback

### ISSUE THREAD

**Issue by germanferrero** [lingui/js-lingui#1167]
**Is your feature request related to a problem? Please describe.**
TLDR; Make plurals fallback to those of locale fallback.

We are migrating to Lingui for [our preact web application](https://github.com/germanferrero/lime-app/tree/a509fbb760953d46d869468415b4586ccbbbf438).
We choosed `@lingui/detect-locale` to detect the browser locale with `fromNavigator`. But, we only have full support for english (`en`) , spanish (`es`) and portuguese (`pt`). In any other case, we would like to fallback to `en` locale, and use plurals rules for that locale too.

At [.linguirc](https://github.com/germanferrero/lime-app/blob/a509fbb760953d46d869468415b4586ccbbbf438/.linguirc) we set up the following, in order to accomplish this:
```javascript
{
...,
"fallbackLocales": {
      default: "en"
   }
}
```

Now, when we a user with japanese (`ja`) locale use the app, we get this error message in the console:
```
Plurals for locale ja aren't loaded. Use i18n.loadLocaleData method to load plurals for specific locale. Using other plural rule as a fallback.
```

And effectively "other plural" rule is always used.
   
**Describe proposed solution**
This issue is related to https://github.com/lingui/js-lingui/issues/798.
But that one was considered closed by falling back to the other plural rule always.
I think it would be better, and more intuitive to fallback both locale and plural rules to the default fallback locale. 

**Additional context**
Other non so user-friendly experience with fallback locales that we have:
- If the user's browser is en-US for instance, by default fallback to its CLDR parent locale `en`, but again the plurals rules do not fallback to `en` plural rules.

**Describe alternatives you've considered**
We ended up trimming the "country" specification from the browser locale before locale activation to avoid plural rules errors regarding CLDR parent locales. But of course this is not what we want to do with our i18n.

**Comment by germanferrero** [comment]
> Hey, maybe stale bot shouldn't be marking this as stale, don't you think?

...

**Comment by thekip** [comment]
`fallbackLocales` settings in `lingui.config.js` has effect only on compilation of catalog. It is not intended to affect you runtime logic anyhow. 

It's up to host application how to process situations when user asked a version which you do not support. 
In your case, it's simple:
```ts
const availableLanguages = ["en", "es", "pt"]; 
const locale = availableLanguages.includes(requestedLocale) ? requestedLocale : "en";

i18n.activate(locale);
```
